### PR TITLE
New exporter script for maplab.

### PR DESCRIPTION
**Pull request by fabianbl**
Can convert a `camchain-imucam.yaml` + `imu.yaml` file to a `sensors.yaml` file and vice versa.

Usage:
```bash
# This will create a sensors.yaml file.
kalibr_maplab_config --cam camchain-imucam.yaml --imu imu.yaml

# This will create a camchain-imucam.yaml file.
kalibr_maplab_config --cam sensors.yaml
```

### CODE REVIEW

**Comment by NikolausDemmel** [comment]
Side question: What is maplab?

**Comment by fabianbl** [comment]
@NikolausDemmel It's our visual mapping framework we use at ASL. It uses a different yaml file than the one from kalibr.

**Comment by NikolausDemmel** [comment]
Ah, thanks for the info. I'm guessing it is internal, i.e. not open source?

**Comment by fabianbl** [comment]
Yes, it's internal.

File: aslam_offline_calibration/kalibr/python/exporters/kalibr_maplab_config [hunk 1]
```diff
@@ -0,0 +1,396 @@
...
+        self.id = imu_dict['id']
+        self.update_rate = 200.0
+
+        # Gyro noise density. [rad/s*1/sqrt(Hz)]
+        self.gyro_noise_density = imu_dict['sigmas']['gyro_noise_density']
+        # Accelerometer noise density. [m/s^2*1/sqrt(Hz)]
+        self.acc_noise_density = imu_dict['sigmas']['acc_noise_density']
+        # Gyro bias random walk. [rad/s^2*1/sqrt(Hz)]
+        self.gyro_random_walk = imu_dict['sigmas'][
+            'gyro_bias_random_walk_noise_density']
```
Comment (schneith) [inline aslam_offline_calibration/kalibr/python/exporters/kalibr_maplab_config]: gyro_random_walk

**Pull request merged**